service/arm_templates/main.json

```diff
@@ -366,7 +325,7 @@
         "parameters": {
           "dnsZone": { "value": "[parameters('dnsZones').etcd]" },
           "prefix": { "value": "[concat('etcd.', parameters('clusterID'))]" },
-          "ipAddress": { "value": "[reference('etcd_load_balancer_setup').outputs.ipAddress.value]" }
+          "ipAddress": { "value": "[reference('master_nodes_setup_1').outputs.ipAddress.value]" }
```

Comment:
This is a bit dirty. I assume there is only a single master.

If you prefer, I can try to find a way to set a round robin DNS in case there is more than one master nodes.

Reply:
Fine for now.